plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/WebXmlValidatorPluginTest.java

```diff
@@ -48,10 +46,12 @@
   private IJavaProject javaProject;
   private IResource resource;
   @Rule public TestProjectCreator projectCreator = new TestProjectCreator().withFacets(
-      JavaFacet.VERSION_1_7, WebFacetUtils.WEB_25, AppEngineStandardFacet.JRE7);
+      JavaFacet.VERSION_1_7);
 
-  @Before
-  public void setUp() throws CoreException {
+  private void setUpProject(boolean dynamicWebProject) throws CoreException {
```

Comment:
Perhaps we'd be better separating the tests into two classes, one with tests requiring only a plain project, the other for tests requiring the DWP.

Reply:
Let me try that.

Reply:
Done!